spec/models/manageiq/providers/kubernetes/container_manager/container_group_spec.rb

```diff
@@ -11,4 +11,33 @@
     )
     expect(group.container_images.count).to eq(1)
   end
+
+  # check https://bugzilla.redhat.com/show_bug.cgi?id=1406770
+  it "has container volumes as parent" do
+    group = FactoryGirl.create(
+      :container_group,
+      :name => "group",
+    )
+
+    ems = FactoryGirl.create(
+      :ems_kubernetes,
+      :id   => group.id,
+      :name => "ems"
+    )
+
+    FactoryGirl.create(
+      :container_volume,
+      :name   => "group_child",
+      :parent => group
+    )
+
+    FactoryGirl.create(
+      :persistent_volume,
+      :name   => "ems_child",
+      :parent => ems
+    )
+
+    expect(group.container_volumes.count).to eq(1)
+    expect(group.container_volumes.first.name).to eq("group_child")
```

Comment:
I'd feel better testing all directions:

1. pod.container_volumes contains only the container_volume.
2. container_volume.parent is pod.
3. ems.persistent_volumes contains only the persistent_volume. 
4. persistent_volume.parent is ems.

and then same things when instead of creating via FactoryGirl.create with :parent,
you do `group.container_volumes.create(...)` and `ems.persistent_volumes.create(...)`.